server/templates/nl_interface_data.html

```diff
@@ -0,0 +1,40 @@
+{#
+  Copyright 2023 Google LLC
+ 
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ #}
+ {%- extends BASE_HTML -%}
+ 
+ {% set is_hide_full_footer = true %}
+ {% set is_hide_sub_footer = true %}
+ {% set main_id = 'nl-interface' %}
+ {% set page_id = 'nl-interface-page' %}
+ {% set title = 'NL Explorer' %}
+ {% set subpage_title = 'NL Explorer' %}
+ {% set subpage_url = url_for('nl.page') %}
+ 
+ {% block head %}
+ <link rel="stylesheet" href={{url_for('static', filename='css/nl_interface.min.css', t=config['VERSION'])}} >
+ <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
do we need to load these fonts?

Reply:
yes because we're still showing the same nl interface which uses material icons

Reply:
but do the charts need these fonts, since here we only need the svg?

Reply:
I don't think the charts will need the font, but the page will look broken? (in the case that someone hits the /nl/data/... address, but I guess that's ok)

Reply:
This endpoint is only meant for selenium to get svg data, so no need to worry about the look of other parts.

Reply:
updated